LFPy/network.py

```diff
@@ -947,6 +947,10 @@ def simulate(self, electrode=None, rec_imem=False, rec_vmem=False,
         except AssertionError:
             raise AssertionError('rec_pop_contributions can not be True when electrode is None')
 
+        # raise an Exception if variable_dt == True and dt > 1E-8
+        if (variable_dt == True) and (self.dt > 1E-8):
```

Comment:
maybe if `variable_dt` is True we can overwrite `dt`?

Reply:
I prefer to blame it on the user :)
I think it is more explicit if the user actually have to set a dt which is small in case he/she wants to use variable_dt.

Reply:
Ok :)